app/controllers/pieces_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PiecesController < ApplicationController
+  attr_accessor :captured
```

Comment:
No this won't work either.  Sorry I wasn't clear.  you want attr_accessor to be in the model.  This controller action still has no way of getting the captured value from the piece model.